docs/sources/community/lids/_index.md

```diff
@@ -2,9 +2,8 @@
 title: Loki Improvement Documents (LIDs)
 description: Loki Improvement Documents (LIDs)
 aliases: 
-- /docs/loki/latest/lids
-- /docs/loki/latest/community/lids
-weight: 1400
+- ../../lids/
```

Comment:
I believe the intention is to redirect the page /docs/loki/latest/lids/, so the alias is: 
```suggestion
- ../lids/
```